packages/grafana-runtime/src/utils/plugin.ts

```diff
@@ -31,12 +31,13 @@ export const SystemJS = window.System;
  * @param options - plugin styling for light and dark theme.
  * @public
  */
-export async function loadPluginCss(options: PluginCssOptions): Promise<any> {
+export async function loadPluginCss(options: PluginCssOptions): Promise<System.Module> {
   try {
     const cssPath = config.bootData.user.theme === 'light' ? options.light : options.dark;
-    return await SystemJS.import(cssPath);
+    return SystemJS.import(cssPath);
   } catch (err) {
     console.error(err);
+    return Promise.reject(err);
   }
```

Comment:
This will change the behaviour of the try-catch block. If the `SystemJS.import(..)` will throw an error it won't be captured by the try catch. If will need to be handled outside of this function.